app/settings.py

```diff
@@ -0,0 +1,147 @@
+"""
+Django settings for app project.
+
+Generated by 'django-admin startproject' using Django 2.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '5f^ouu_s%s@@(476pwmglzanj58a#eiv)p&v3x7h6_cs5+6@#c'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
Probably also makes sense to have this off by default and turn it on with an environment variable for local runs (unless Django already provides a more convenient way).

Reply:
Done that!. It'll be set to false if the environment is prod, and will be true otherwise!